ext/standard/basic_functions.stub.php

```diff
@@ -1087,11 +1086,8 @@ function exec(string $command, &$output = null, &$result_code = null): string|fa
  */
 function system(string $command, &$result_code = null): string|false {}
 
-/**
- * @param int $result_code
- * @return false|null
- */
-function passthru(string $command, &$result_code = null): ?bool {}
+/** @param int $result_code */
+function passthru(string $command, &$result_code = null): false|null {}
```

Comment:
It doesn't look like `passthru` returns `false` anymore, but maybe I missed it?

Reply:
It does look that way but it is a couple of nested code deep when it triggers an E_WARNING because it couldn't fork the command, it is however the only one and that also affects exec()